DirectEmployers/MyJobs
# [PD - 2818] Adding the feedback button to the NUO beta

Task was to add a leave feedback button to the NUO UI.

To Test:
Pull down repository and go to NUO in the beta tab. You will see on the right hand sidebar at the bottom of the navigation there is a link that opens a separate window to the feedback form that says Leave Us Feedback.
